sklearn/ensemble/_bagging.py

```diff
@@ -577,8 +577,9 @@ class BaggingClassifier(ClassifierMixin, BaseBagging):
     .. [4] G. Louppe and P. Geurts, "Ensembles on Random Patches", Machine
            Learning and Knowledge Discovery in Databases, 346-361, 2012.
     """
+    @_deprecate_positional_args
     def __init__(self,
-                 base_estimator=None,
+                 base_estimator=None, *,
                  n_estimators=10,
```

Comment:
I would consider accepting n_estimators as positional. WDYT? `BaggingClassifier(DecisionTreeClassifier(), 100)`

Reply:
Fair enough, it's just that I decided to put n_estimator as kw-only in the other ones, and made sense to do the same here.